system_test/helpers/aest_docker.erl

```diff
@@ -450,6 +448,19 @@ attempt_epoch_stop(#{container_id := ID, hostname := Name} = NodeState, Timeout)
     end,
     ok.
 
+%% Sometime the stop command do not succeed...
+retry_epoch_stop(_NodeState, _ID, _Cmd, _Opts, 0) -> error;
+retry_epoch_stop(NodeState, ID, Cmd, Opts, Retry) ->
+    #{container_id := ID, hostname := Name} = NodeState,
+    case aest_docker_api:exec(ID, Cmd, Opts) of
+        {ok, 137, <<"ok\r\n">>} -> ok;
+        {ok, Status, Res} ->
+            CmdStr = lists:join($ , Cmd),
```

Comment:
This is no my code, I just refactored it. This is a space character, if I quote it with `'` it would become an atom and if I quote it with `"` it would become a list.